nest/core/decorators/controller.py

```diff
@@ -24,7 +24,9 @@ def Controller(prefix: Optional[str] = None, tag: Optional[str] = None):
     route_prefix = process_prefix(prefix, tag)
 
     def wrapper(cls: Type) -> Type[ClassBasedView]:
-        router = APIRouter(tags=[tag] if tag else None)
+        tags = [tag] if tag else None
+        # TODO: replace with factory
+        router: Router = FastAPIRouter(tags=tags)
```

Comment:
Need to think of a way to remove router intilization. 
Could be done by creating abs for controller and make instace for fastapi/others. 

instead of 
```python
from nest.core.decrator import Controller
from nest.decrator.fastapi import Controller
```